ssl/ssl_ciph.c

```diff
@@ -498,12 +499,12 @@ static void do_load_builtin_compressions(void)
         }
     }
     CRYPTO_mem_ctrl(CRYPTO_MEM_CHECK_ENABLE);
+    return 1;
```

Comment:
I would say that the failure to allocate a compression method isn't critical enough to warrant a different return value.  Are you of a different opinion?